src/main/java/marquez/common/Utils.java

```diff
@@ -54,7 +56,10 @@ private Utils() {}
   public static ObjectMapper newObjectMapper() {
     final ObjectMapper mapper = Jackson.newObjectMapper();
     mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
-    mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+    mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    mapper.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
+    mapper.registerModule(new JavaTimeModule());
```

Comment:
We can omit registering `JavaTimeModule` as the `ObjectMapper` instance returned by `Jackson.newObjectMapper()`  configures it for us (see https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jackson/src/main/java/io/dropwizard/jackson/Jackson.java#L70).